client/src/features/Catalogue/components/CatalogFilterDuration/CatalogFilterDuration.js

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect } from 'react';
+import { shallowEqual, useDispatch, useSelector } from 'react-redux';
+import Grid from '@mui/material/Unstable_Grid2';
+import { FilterAccordion, OutlinedToggleButton } from '..';
+import { getProducts } from '../../../../store/slices/catalogueSlice/catalogueSlice';
+
+
+const CatalogFilterDuration = () => {
+  const dispatch = useDispatch();
+  const products = useSelector((state) => state.catalogue.products, shallowEqual);
+
+  const sorting = () => {
+    const duration = [...new Set(products.map((el) => el.duration))];
```

Comment:
what is this? and why?

```suggestion
    const duration = products.map((el) => el.duration);
```

why not just write it in the above form?

Reply:
This way I get only unique values ​​for the duration of the tours, that is, without repeating the same